src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/product/create.yaml

```diff
@@ -134,17 +136,6 @@ sylius_twig_hooks:
                 template: '@SyliusAdmin/product/form/sections/translations/short_description.html.twig'
                 priority: 0
 
-        'sylius_admin.product.create.content.form.sections.channel_pricing':
-            price:
-                template: '@SyliusAdmin/product/form/sections/channel_pricing/price.html.twig'
-                priority: 200
-            original_price:
-                template: '@SyliusAdmin/product/form/sections/channel_pricing/original_price.html.twig'
-                priority: 100
-            minimum_price:
-                template: '@SyliusAdmin/product/form/sections/channel_pricing/minimum_price.html.twig'
-                priority: 0
-
```

Comment:
I have some doubts about removing existing hooks due to backward compatibility, does maybe anyone have an opinion here? 🤔

Reply:
I would agree -- share templates, not hooks.

Maybe we can request `extends` feature for `sylius/twig-hooks` package :smile: An ability to extend common hook as is, but under current resource specific naming.